wavenet.py

```diff
@@ -108,14 +121,15 @@ def _create_network(self, input_batch):
             # Perform (+) -> ReLU -> 1x1 conv -> ReLU -> 1x1 conv to
             # postprocess the output.
             w1 = tf.Variable(tf.truncated_normal(
-                [1, self.residual_channels, int(self.channels / 2)], stddev=0.3,
+                [1, self.residual_channels, self.channels], stddev=0.3,
```

Comment:
The intermediate tensor could be something else, but the number of hyperparameters is growing out of hand... How about skip_channels is a vector that defines the channels per layer, with optionally deeper convolutions than the 2 layers we have now? See https://github.com/ibab/tensorflow-wavenet/pull/14/files#diff-59f2c602fd3753c87268cb878555ae38R135 for an implementation.